display_list/display_list_utils.cc

```diff
@@ -353,47 +483,34 @@ void DisplayListBoundsCalculator::restore() {
     // Remember a few pieces of information from the current layer info
     // for later processing.
     LayerData* layer_info = layer_infos_.back().get();
-    BoundsAccumulator* outer_accumulator = layer_info->restore_accumulator();
     bool is_unbounded = layer_info->is_unbounded();
 
     // Before we pop_back we will get the current layer bounds from the
     // current accumulator and adjust ot as required based on the filter.
-    SkRect layer_bounds = accumulator_->bounds();
+    // SkRect layer_bounds = accumulator_->bounds();
```

Comment:
Please remove this line of commented code :-)

Reply:
Good catch! Done.